torchvision/datasets/hmdb51.py

```diff
@@ -110,6 +117,10 @@ def __getitem__(self, idx):
         label = self.samples[self.indices[video_idx]][1]
 
         if self.transform is not None:
-            video = self.transform(video)
+            transformed_video = []
+            for counter, image in enumerate(video):
+                image = self.transform(image)
+                transformed_video.append(image)
+            video = torch.stack(transformed_video)
```

Comment:
Those are unrelated changes that shouldn't be handled this way.
Instead, we are working on video transforms that can be applied to a video clip right away (they are currently private functions because their API will change and the default transforms will natively support video clips, but the private functions are in https://github.com/pytorch/vision/blob/master/torchvision/transforms/_transforms_video.py)